### PR TITLE
📖 add CAPO to cluster-api book

### PR DESCRIPTION
**Pull request by sbueringer**
**What this PR does / why we need it**:
This PR adds CAPO to the ClusterAPI book.

### CODE REVIEW

**Comment by sbueringer** [comment]
/hold

until the following PRs are merged:
* [ ] Release CAPO v0.2.0 Image: https://github.com/kubernetes/k8s.io/pull/433
* [ ] Finalize CAPO docs & examples https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/488

File: docs/book/src/tasks/installation.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Cluster API requires an existing kubernetes cluster accessible via kubectl, choo
 
 1. **Kind**
 
-{{#tabs name:"kind-cluster" tabs:"AWS,Docker,vSphere"}}
+{{#tabs name:"kind-cluster" tabs:"AWS,Docker,vSphere,OpenStack"}}
```
Comment (vincepri) [inline docs/book/src/tasks/installation.md]: It seems that we're duplicating instructions for all except Docker? If so, we should probably collapse them into one

Comment (sbueringer) [reply docs/book/src/tasks/installation.md]: Good idea. Done.

Comment (randomvariable) [reply docs/book/src/tasks/installation.md]: Looking good.

/lgtm

**Comment by vincepri** [comment]
@sbueringer Can you squash to 1 commit?

**Comment by sbueringer** [comment]
> @sbueringer Can you squash to 1 commit?

Of course. Done

**Comment by sbueringer** [comment]
/hold cancel
PRs in k8s.io and CAPO have been merged

**Comment by sbueringer** [comment]
@detiber is it possible to get this published in https://cluster-api.sigs.k8s.io/user/quick-start.html?

**Comment by vincepri** [comment]
@sbueringer The book is published from `release-0.2`, can you open a PR to backport this commit (cherry-pick)?

**Comment by sbueringer** [comment]
> @sbueringer The book is published from `release-0.2`, can you open a PR to backport this commit (cherry-pick)?

Yes, done > https://github.com/kubernetes-sigs/cluster-api/pull/1680

**Pull request merged**